nixos/modules/services/desktops/pipewire/pipewire-media-session.nix

```diff
@@ -78,6 +80,15 @@ in {
           default = {};
         };
 
+        bluez-hardware = mkOption {
+          type = json.type;
+          description = ''
+            Configuration for the bluez5 hardware database. For details see
+            https://gitlab.freedesktop.org/pipewire/pipewire/-/blob/${cfg.package.version}/src/daemon/media-session.d/bluez-hardware.conf
+          '';
+          default = {};
+        };
+
```

Comment:
Like I mentioned in my comment, please remove this option but keep the file in /etc. Other than that everything looks good & works on my machine as it should.